opentelemetry-sdk/src/trace/provider.rs

```diff
@@ -39,9 +58,14 @@ impl Drop for TracerProviderInner {
 }
 
 /// Creator and registry of named [`Tracer`] instances.
+///
+/// `TracerProvider` is lightweight container holding pointers to `SpanProcessor` and other components.
+/// Cloning and dropping them will not stop the span processing. To stop span processing, users
+/// must either call `shutdown` method explicitly, or drop every clone of `TracerProvider`.
 #[derive(Clone, Debug)]
 pub struct TracerProvider {
     inner: Arc<TracerProviderInner>,
+    is_shutdown: Arc<AtomicBool>,
```

Comment:
Is there any benefit to creating another Arc pointer just for shutdown? Could we reuse the existing Arc pointer inner to also hold is_shutdown inside it?